src/core/services/theming/theming.js

```diff
@@ -574,6 +574,8 @@ function ThemingProvider($mdColorPalette, $$mdMetaProvider) {
    * @description
    * Service that makes an element apply theming related <b>classes</b> to itself.
    *
+   * For more Information on the `xHues` objects, their default values, as well as valid hue values, please visit <a ng-href="Theming/03_configuring_a_theme#specifying-custom-hues-for-color-intentions">the custom hues section of Configuring a Theme</a>.
```

Comment:
OK, out of respect for your limited time, I will hold off on some feedback, but I do want to fix one thing in this line.

Can you please change `Information on the `xHues` objects` to `information on the hue objects`?

Reply:
Done :-)